data_steward/cdr_cleaner/cleaning_rules/date_unshift_cope_responses.py

```diff
@@ -29,24 +29,29 @@
 
 OBSERVATION_EXT = f'{OBSERVATION}_ext'
 
+SURVEY_VERSION_CPT_ID_QUERY = JINJA_ENV.from_string("""
+SELECT
+DISTINCT e.survey_version_concept_id
+FROM `{{project_id}}.{{dataset_id}}.{{observation_ext_table}}` e
+JOIN `{{project_id}}.{{dataset_id}}.concept` c
+ON c.concept_id = e.survey_version_concept_id
```

Comment:
Add a condition here to make sure only cope/covid are affected
```suggestion
ON c.concept_id = e.survey_version_concept_id
WHERE REGEXP_CONTAINS(c.concept_name, r'(COPE)|(COVID)')
```